docs/how-to/custom-env.txt

```diff
@@ -105,8 +105,8 @@ This can be configured in your experiment configuration like below:
 
    environment:
      image:
-       gpu: "determinedai/environments:cuda-10.1-pytorch-1.4-tf-2.1-gpu-0.4.0"
-       cpu: "determinedai/environments:py-3.6.9-pytorch-1.4-tf-2.1-cpu-0.4.0"
+       gpu: "determinedai/environments:cuda-10.1-pytorch-1.4-tf-2.2-gpu-0.4.0"
```

Comment:
blocking: gotta change these two lines as well!

https://github.com/determined-ai/determined/blob/cb8f4a82b1ac44850aa63fdc7498beacd741321e/docs/how-to/custom-env.txt#L93
https://github.com/determined-ai/determined/blob/cb8f4a82b1ac44850aa63fdc7498beacd741321e/docs/how-to/custom-env.txt#L99

Reply:
Good call updated.